packages/components/src/ecosystem_bar/index.jsx

```diff
@@ -0,0 +1,42 @@
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import { useRef } from 'react';
+import { useOnceSwipeTracking, useScrolled } from '@/src/ecosystem_bar/hooks';
+import styles from './index.module.scss';
+
+const EcosystemBar = ({ items, onFirstSwipe }) => {
+  const navRef = useRef(null);
+  const scrolled = useScrolled(navRef);
+
+  useOnceSwipeTracking(navRef, onFirstSwipe);
+
+  return (
+    <ul className={clsx(styles.root, { [styles.scrolled]: scrolled })} ref={navRef}>
+      {items.map((item) => (
+        <li key={item.key} className={styles.item}>
+          <a
+            className={clsx(styles.link, { [styles.isActive]: (item.key === item.project) })}
```

Comment:
I think having `project: 'core'` in each menu item is not cool, let's extract `project` to a separate prop?

do we actually need to know what project it is? or we can just add `items[].active: true` to some of the menu items and here change the condition to 
```
[styles.isActive]: item.active
```
?

Reply:
Since we now are creating the items for each project individually we can also set `isActive` key while doing so. I think I'd prefer that, also the solves the passing of another project prop.